DM_2/config/sequelizeConfig.js

```diff
@@ -0,0 +1,13 @@
+const {Sequelize} = require('sequelize');
+
+const sequelize = new Sequelize('demo2_db', 'postgres', 'qwerty', {
+  host: 'localhost',
+  dialect: 'postgres',
+  define: {
+    timestamps: false
+  }
+});
```

Comment:
Since you added `.env`, you can use environmental variables here for `DB_NAME`, `USERNAME`, `PASSWORD`, `HOST`, `DIALECT`.

Reply:
done